format/index.md

```diff
@@ -93,16 +93,22 @@ Primary data **MUST** be either:
 
 For example, the following primary data is a single resource object:
 
-```javascript
+```json
 {
   "data": {
     "type": "articles",
     "id": "1",
     "attributes": {
-      // ... this article's attributes
+      "title": "JSON API paints my bikeshed!"
```

Comment:
Since comments are not allowed in JSON, I've switched to including the entire object.